src/main/java/org/spongepowered/api/attributes/Attribute.java

```diff
@@ -0,0 +1,36 @@
+package org.spongepowered.api.attributes;
+
+public interface Attribute {
+	
+	/**
+	 * Gets the ID of this attribute as a string.
+	 * @return The ID of this attribute as a string.
```

Comment:
Please insert a new line between the doc and the return. Applies to the rest of the javadocs.